code/modules/reagents/reagents/other.dm

```diff
@@ -505,3 +505,23 @@
 /datum/reagent/other/rejuvenating_agent/touch_obj(obj/O)
 	if(istype(O))
 		O.make_young()
+
+/datum/reagent/resuscitator
+	name = "Resuscitator"
+	id = "resuscitator"
+	description = "Incredibly rare cardiac stimulant."
+	reagent_state = LIQUID
+	color = "#A0522D"
+	metabolism = 0.5
+	overdose = REAGENTS_OVERDOSE
+	scannable = 1
+	affects_dead = 1
+
+/datum/reagent/resuscitator/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
```

Comment:
If I'm looking at this right, it'd only take 0.5, maybe even less, to resuscitate somebody.

Reply:
I can change it if you insist. But it also removes oxydamage so resuscitation can go easily.

Reply:
Molitor-Riedel Enricher make 1 unit of resuscitator per 4 nutriments.

Reply:
Done